torch/csrc/jit/script/module.h

```diff
@@ -149,12 +149,9 @@ struct Method {
     return *this;
   }
 
-  const FunctionSchema& getSchema() const {
-    AT_ASSERT(schema != nullptr);
-    return *schema;
-  }
+  const FunctionSchema& getSchema() const;
 
-  std::string prettyPrintSchema() const {
+  std::string pretty_print_schema() const {
```

Comment:
Why the change in naming convention? We usually use camel case in the JIT, right?

Reply:
module.h is bound to python. Its other methods use python convention, so I am trying to incrementally make it more consistent. There are other changes but they were more invasive so I didn't do it yet.